server/root.ts

```diff
@@ -461,20 +461,32 @@ const root = {
         currentUser.chats,
         params.fromUserEmail
       )
+      const fromUserLimited = {
```

Comment:
had to debug this. before, this code wasn't properly updating user hangout requests if they already knew each other. should be all fixed.